script.js

```diff
@@ -0,0 +1,364 @@
+initializeCalculator();
+function initializeCalculator() {
```

Comment:
I'd call this function `Calculator`, since it contains everything calculator-related and does not only deal with initialising the calculator. You could also consider using a [Javascript class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)